blockchain/hedera.go

```diff
@@ -57,21 +72,35 @@ func (hSubscr hederaSubscriber) SubscribeToEvents(channel chan<- subscriber.Even
 	//todo map somehow task id from memo to a different jobspec? is that possible? what is jobID in hederaSubscription?
 	//todo check if payment was okay
 	//todo create tests - hedera_test.go & also elsewhere where we changed the code + db migration
+	//todo remove last transaction timestamp from hedera_subscription database object if we're not going to save it
 
```

Comment:
I think we've already implemented these //TODOs and can delete them, right?

Reply:
removed